src/index.js

```diff
@@ -0,0 +1,203 @@
+    if (selection === 'sunny') {
+        state.skyAccessible = accessibleText.backgroundSunny;
+        state.backgroundImage = images.backgroundSunny;
+    } else if (selection === 'cloudy') {
+        state.skyAccessible = accessibleText.backgroundCloudy;
+        state.backgroundImage = images.backgroundCloudy;
+    } else if (selection === 'raining') {
+        state.skyAccessible = accessibleText.backgroundRaining;
+        state.backgroundImage = images.backgroundRaining;
+    } else if (selection === 'snowing') {
+        state.skyAccessible = accessibleText.backgroundSnowing;
+        state.backgroundImage = images.backgroundSnowing;
+    }
+    skyAccessible.textContent = state.skyAccessible;
+    skyBackground.style.backgroundImage = `url('${state.backgroundImage}')`;
+};
+
+const changeCity = () => {
+    const city = document.getElementById('city');
+    const cityName = document.getElementById('cityName');
+    cityName.textContent = city.value;
+};
+
+const reset = () => {
+    const city = document.getElementById('city');
+    const cityName = document.getElementById('cityName');
+    const temp = document.getElementById('temp');
+    const foregroundAccessible = document.getElementById(
+        'weatherGardenAccessibleForeground'
+    );
+    const skyAccessible = document.getElementById('weatherGardenAccessibleSky');
+    const skyBackground = document.getElementById('sky');
+    const foreground = document.getElementById('weatherGarden');
+    state.city = 'Baltimore';
+    state.temp = 70;
+    state.tempColour = 'black';
+    state.foregroundAccessible = accessibleText.foregroundSpring;
+    state.skyAccessible = accessibleText.backgroundSunny;
+    state.backgroundImage = images.backgroundSunny;
+    state.foreground = images.foregroundSpring;
+    city.value = state.city;
+    cityName.textContent = state.city;
+    temp.textContent = `${state.temp}°F`;
+    temp.style.color = state.tempColour;
+    foregroundAccessible.textContent = state.foregroundAccessible;
+    skyAccessible.textContent = state.skyAccessible;
+    skyBackground.style.backgroundImage = `url('${state.backgroundImage}')`;
+    foreground.src = state.foreground;
+};
+
+const registerEventHandlers = () => {
+    const up = document.getElementById('up');
+    up.addEventListener('click', increaseTemp);
+    const down = document.getElementById('down');
+    down.addEventListener('click', decreaseTemp);
+    const sky = document.getElementById('sky-select');
+    sky.addEventListener('change', changeSky);
+    const city = document.getElementById('city');
+    city.addEventListener('change', changeCity);
+    const getWeatherButton = document.getElementById('getWeatherButton');
+    getWeatherButton.addEventListener('click', getLocation);
+    const resetButton = document.getElementById('resetButton');
+    resetButton.addEventListener('click', reset);
+};
+
+document.addEventListener('DOMContentLoaded', registerEventHandlers);
```

Comment:
It's easy to overlook this line nestled between functions. I recommend moving it to the very bottom to keep functions together and separate from code that we immediately call and execute.